testing/dynamodb/integration_test.go

```diff
@@ -228,17 +265,24 @@ func checkRow(ctx context.Context, t *testing.T, client *spanner.Client) {
 			t.Fatal(err)
 			break
 		}
-		var year, title, plot string
-		if err := row.Columns(&year, &title, &plot); err != nil {
+		var AttrInt, AttrFloat big.Rat
```

Comment:
Why AttrInt and AttrFloat are big.Rat instead of int64 and float64?

Reply:
I forgot this. The `Number` type in dynamodb will be converted to `Numeric` or `String` in spanner.